lib/src/int64.dart

```diff
@@ -462,9 +462,12 @@ class Int64 implements IntX {
   }
 
   @override
-  int compareTo(other) => _compareTo(other);
+  int compareTo(Object? other) {
+    if (other == null) throw ArgumentError(null);
```

Comment:
This should be `ArgumentError.notNull`, not `ArgumentError(null)`, but there is also a utility for this.

```suggestion
    ArgumentError.checkNotNull(other);
```

Or if we want the name to show up in the error

```dart
    ArgumentError.checkNotNull(other, 'other');
```